### PR TITLE
BackEnd added

### PR DESCRIPTION
**Pull request by mike927**
stable app backend

### CODE REVIEW

File: BackEnd/.gitignore [hunk 1]
```diff
@@ -15,3 +15,7 @@
 /log/*
 !/log/.keep
 /tmp
+
+config/settings.local.yml
+config/settings/*.local.yml
+config/environments/*.local.yml
```
Comment (spidzio) [inline BackEnd/.gitignore]: Why is it ignored ?

**Comment by spidzio** [comment]
You can remove all `.keep` files besides `log/.keep`

File: BackEnd/.gitignore (hunk 1 above)
Comment (dropsrobs) [inline BackEnd/.gitignore]: I'm sure they were automatically ignored by gem. I didn't notice that.

Comment (spidzio) [inline BackEnd/.gitignore]: That's strange. Lets leave it then.

Comment (dropsrobs) [inline BackEnd/.gitignore]: Those files are for developers that want to make local-only configurations for they own use. That's why they were ignored by gem.

**Comment by spidzio** [comment]
You are not controlling what you are commiting and pushing.
The latest commit should be called 'Removed .keep files and added .idea files' :-)
Please remove .idea files and ignore them globally in you .gitconfig

**Comment by spidzio** [comment]
:+1:

**Pull request merged**